tests/test_classifier_interface.py

```diff
@@ -47,19 +49,38 @@
 inference_precision_methods = ["auto", "autocast", torch.float64, torch.float16]
 remove_outliers_stds = [None, 12]
 estimators = [1, 2]
+model_paths = ModelSource.get_classifier_v2().filenames
+
+primary_model = model_paths[0]
```

Comment:
the first one is the one we get by default right? is it safer to check it has the expected name so the test behavior doesn't change if the ordering changes? Not sure